content/modules/python/pages/writing-functions.adoc

```diff
@@ -2,7 +2,7 @@
 
 == Introduction
 
-While functions are more emphasized in object-oriented languages, the fundamental idea of functions is packing several actions into one unit that you yourself define, which is something that applies to all programming languages. When we're dealing with longer and more complicated projects, writing Python functions is crucial for making the code readable.
+The fundamental idea of functions is packing several actions into one unit that you define. When we're dealing with longer and more complicated projects, writing Python functions is crucial for making the code readable and a reasonable length.
```

Comment:
At their core, functions pack several actions into one defined unit. When we deal with more complicated projects, writing Python functions is critical to creating code that is readable and/also has a reasonable length.